### PR TITLE
Set permissions for GitHub actions

### PR DESCRIPTION
**Pull request by naveensrinivasan**
- Included permissions for the action. https://github.com/ossf/scorecard/blob/main/docs/checks.md#token-permissions

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

related to this https://github.com/npm/cli/pull/4602

### CODE REVIEW

**Comment by naveensrinivasan** [comment]
@wraithgar 👀  please

**Comment by wraithgar** [comment]
The benchmark PRs need more access than that, I believe.

**Comment by naveensrinivasan** [comment]
> The benchmark PRs need more access than that, I believe.

OK, let me know which you think aren't OK, I can change them back to original if that works for you.

File: .github/workflows/benchmark-comment.yml [hunk 1]
```diff
@@ -5,8 +5,13 @@ on:
   issue_comment:
     types: [created, edited]
 
+permissions:
+  contents: read
```
Comment (wraithgar) [inline .github/workflows/benchmark-comment.yml]: This will break this action

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -6,8 +6,13 @@ on:
     branches:
       - "**"
 
+permissions:
+  contents: read
```
Comment (wraithgar) [inline .github/workflows/benchmark.yml]: This will break this action

File: .github/workflows/release-please-libnpmaccess.yml [hunk 3]
```diff
@@ -10,8 +10,13 @@ on:
       - release-next
       - latest
 
+permissions:
+  contents: read
```
Comment (wraithgar) [inline .github/workflows/release-please-libnpmaccess.yml]: Not sure what the purpose of setting permissions that immediately get overwritten in a single job is.

File: .github/workflows/benchmark-comment.yml (hunk 1 above)
Comment (naveensrinivasan) [reply .github/workflows/benchmark-comment.yml]: Removed it.

File: .github/workflows/benchmark.yml (hunk 2 above)
Comment (naveensrinivasan) [reply .github/workflows/benchmark.yml]: Removed it.

File: .github/workflows/release-please-libnpmaccess.yml (hunk 3 above)
Comment (naveensrinivasan) [reply .github/workflows/release-please-libnpmaccess.yml]: Removed it. Thanks

**Comment by wraithgar** [comment]
All of the release-please PRs are going to need the same consolidation of permissions. I only commented in one of them.

**Comment by naveensrinivasan** [comment]
> All of the release-please PRs are going to need the same consolidation of permissions. I only commented in one of them.

So removed the permissions in release-please based on your comment.

**Comment by wraithgar** [comment]
We're going to have to fix this in template-oss, not here.  Almost all of these files (at least the ones we know we want to change) will be overwritten by template-oss when we run it in the cli next.

**Comment by wraithgar** [comment]
All of these files are managed by template-oss so fixes need to happen there.

**Pull request closed without merging**